fab/fabfile.py

```diff
@@ -367,6 +367,14 @@ def install_packages():
 
 
 @task
+@roles(ROLES_TOUCHFORMS)
+def install_npm_packages():
+    """Install required NPM packages for server"""
+    with cd(os.path.join(env.code_root, 'submodules/touchforms-src/touchforms')):
+        sudo("npm install")
```

Comment:
excellent! i'm glad we are in accordance. i will push forward with this, but will test this deploy out on staging first. it's going to take a bit of coordination